addon-test-support/@ember/test-helpers/test-metadata.ts

```diff
@@ -0,0 +1,27 @@
+import { BaseContext } from './setup-context';
+
+export interface ITestMetadata {
```

Comment:
It mildly disturbs me that all other interfaces in this project don't use the 'I' prefix, but I also find it very irritating that you can't define an interface and class of the same name (for obvious reasons), which forces you to come up with some awful name just to accommodate the mismatch.